sda-commons-dependencies/build.gradle

```diff
@@ -65,8 +74,9 @@ dependencies {
     api "org.jboss.logging:jboss-logging:3.5.3.Final", {
       because "conflict between org.hibernate.validator:hibernate-validator and org.jboss.weld.se:weld-se-core"
     }
-    api "org.yaml:snakeyaml:1.33", {
+    api "org.yaml:snakeyaml:2.0", {
       because "conflict between com.github.ftrossbach:club-topicana-core and com.github.database-rider:rider-core and com.fasterxml.jackson.dataformat:jackson-dataformat-yaml"
```

Comment:
This line still refers to "topicana" but I'm pretty sure we don't use it anymore. We could remove the comment and the version definition further down in another PR.